File: solutions2020/PRA1/ameyag05@gmail.com.py
timesRun = int(input(""))               
answers = []
def checker():
    number = input("")
    number = number.replace(" ", "")
    for i in range (len(number)):
        if number[i] != '1' and number[i] != '4':
            answers.append("NO")
            return None
        elif number[i] == '4' and number[i-1] == '4':
            if i == (len(number)-1):
                break
            elif number[i+1] == '4':
                answers.append("NO") 
                return None
        elif number[0] == '4':
            answers.append("NO")
            return None
    answers.append("YES")
for ogrange in range (timesRun):      
    checker()     
for i in answers:
    print(i)

File: solutions2020/PRB4/ameyag05@gmail.com.java
import random
timesRun = int(input(""))
for timesRunning in range(timesRun):
    function = []
    numberList = []
    finalNum = []
    allFinalNums = []
    for numbers in range(9):
        function.append(random.randrange(0,10,1))
    lenNums = int(input(""))
    number = int(input(""))
    print(function)
    for iterate in str(number):
        numberList.append(int(iterate))
    for checkVal in function:
        i = 0
        if i > len(numberList):
            break
        if checkVal>numberList[i]:
            finalNum.append(checkVal)
        else:
            finalNum.append(numberList[i])
        i = i+1
    allFinalNums.append(finalNum)
print(allFinalNums)

